src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/chain/deal_matched.rs

```diff
@@ -0,0 +1,244 @@
+use cid::Cid;
+use ethabi::param_type::ParamType;
+use ethabi::Token;
+use libp2p_identity::PeerId;
+use marine_rs_sdk::marine;
+
+use crate::chain::chain_data::EventField::{Indexed, NotIndexed};
+use crate::chain::chain_data::{ChainData, EventField, LogParseError};
+use crate::chain::chain_event::ChainEvent;
+use crate::chain::data_tokens::next_opt;
+use crate::chain::u256::U256;
+
+/// Corresponding Solidity type:
+/// ```solidity
+/// struct CIDV1 {
+///     bytes4 prefixes;
+///     bytes32 hash;
+/// }
+///
+/// event ComputePeerMatched(
+///     bytes32 indexed peerId
+///     address deal
+///     bytes32[] patIds
+///     uint dealCreationBlock
+///     CIDV1 appCID
+/// );
+/// ```
+
+const PEER_ID_PREFIX: &[u8] = &[0, 36, 8, 1, 18, 32];
```

Comment:
Where does it come from? I kinda understand but not fully

Reply:
I've added a generic comment. Would be nice to document every byte though.